src/components/logo/Logo.tsx

```diff
@@ -0,0 +1,36 @@
+import Box from "@mui/material/Box";
+import logo from "../../assets/logo.svg";
+import "./Logo.scss";
+
+export const Logo = () => {
+  return (
+    <Box
+      sx={{
+        bgcolor: "#7C5DFA",
+        width: 1,
+        height: "4rem",
```

Comment:
sometimes you use px and sometimes rem. Pick one and stick to it. I'd suggest rem

Reply:
fixed.